daemon/logger/journald/journald.go

```diff
@@ -15,13 +16,22 @@ import (
 const name = "journald"
 
 type journald struct {
-	Jmap map[string]string
+	Jmap    map[string]string
```

Comment:
Sure thing.  It'll be "vars", since they're additional variables and values that we want to add to the journaled log message.